examples/winrm/windows_services.json

```diff
@@ -30,5 +30,12 @@
             "name": "Microsoft Windows",
             "version": "Server 2019"
         }
+    ],
+    "parameters": [
+        {
+            "value_type": "LIST",
+            "name": "servicesList",
+            "label": "Service"
```

Comment:
doesn't look descriptive enough, I'd put this from the comment "List of services to monitor" that's in the actual script

Reply:
Actually the label shows under the input but I agree "Service Name" may show up better